infrastructure/code/bundle.go

```diff
@@ -129,7 +132,7 @@ func (b *Bundle) retrieveAnalysis(ctx context.Context) ([]snyk.Issue, error) {
 	}
 
 	p := progress.NewTracker(false)
```

Comment:
Question: Do we want to use b.progress here now as well?

Reply:
I tested and this doesn't work unfortunately, as ProgressTracker is meant to be transient.
I'd love if you can make it more service-oriented. Currently LS panics when it's reused:
`panic: Called end progress twice. This breaks LSP in Eclipse fix me now and avoid headaches later`.

In fact, I have to revert the last commit, since it panics with the same message, when autofix requested twice.